java.lang.NullPointerException:    at openfoodfacts.github.scrachx.openfood.utils.Utils.HttpClientBuilder

```
30 minutes ago on app version 52
Xiaomi Redmi Note 3 (hennessy), 2048MB RAM, Android 5.0
Report 1 of 1
java.lang.RuntimeException: 
  at android.app.ActivityThread.performStopActivityInner (ActivityThread.java)
  at android.app.ActivityThread.handleStopActivity (ActivityThread.java)
  at android.app.ActivityThread.access$1200 (ActivityThread.java)
  at android.app.ActivityThread$H.handleMessage (ActivityThread.java)
  at android.os.Handler.dispatchMessage (Handler.java)
  at android.os.Looper.loop (Looper.java)
  at android.app.ActivityThread.main (ActivityThread.java)
  at java.lang.reflect.Method.invoke (Method.java)
  at java.lang.reflect.Method.invoke (Method.java)
  at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run (ZygoteInit.java)
  at com.android.internal.os.ZygoteInit.main (ZygoteInit.java)
Caused by: java.lang.NullPointerException: 
  at openfoodfacts.github.scrachx.openfood.utils.Utils.HttpClientBuilder (Utils.java)
  or                     .apply (Utils.java)
  or                     .bold (Utils.java)
  or                     .compressImage (Utils.java)
  or                     .convertKjToKcal (Utils.java)
  or                     .decodeFile (Utils.java)
  or                     .getAppDaoSession (Utils.java)
  or                     .getBitmapFromDrawable (Utils.java)
  or                     .getClickableText (Utils.java)
  or                     .hideKeyboard (Utils.java)
  or                     .isApplicationInstalled (Utils.java)
  or                     .lambda$scan$0$Utils (Utils.java)
  or                     .openTags (Utils.java)
  at openfoodfacts.github.scrachx.openfood.fragments.SaveOfflineSummaryFragment.getProduct (SaveOfflineSummaryFragment.java)
  at openfoodfacts.github.scrachx.openfood.views.SaveProductOfflineActivity.lambda$onRequestPermissionsResult$5$SaveProductOfflineActivity (SaveProductOfflineActivity.java)
  or                     .saveProduct (SaveProductOfflineActivity.java)
  at openfoodfacts.github.scrachx.openfood.views.SaveProductOfflineActivity.onStop (SaveProductOfflineActivity.java)
  at android.app.Instrumentation.callActivityOnStop (Instrumentation.java)
  at android.app.Activity.performStop (Activity.java)
  at android.app.ActivityThread.performStopActivityInner (ActivityThread.java)
  at android.app.ActivityThread.handleStopActivity (ActivityThread.java)
  at android.app.ActivityThread.access$1200 (ActivityThread.java)
  at android.app.ActivityThread$H.handleMessage (ActivityThread.java)
  at android.os.Handler.dispatchMessage (Handler.java)
  at android.os.Looper.loop (Looper.java)
  at android.app.ActivityThread.main (ActivityThread.java)
  at java.lang.reflect.Method.invoke (Method.java)
  at java.lang.reflect.Method.invoke (Method.java)
  at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run (ZygoteInit.java)
  at com.android.internal.os.ZygoteInit.main (ZygoteInit.java)
```

--- a/app/src/main/java/openfoodfacts/github/scrachx/openfood/utils/Utils.java
+++ b/app/src/main/java/openfoodfacts/github/scrachx/openfood/utils/Utils.java
@@ -132,6 +132,9 @@ public class Utils {
     }
 
     public static void hideKeyboard(Activity activity) {
+        if(activity == null)
+            return;
+        
         View view = activity.getCurrentFocus();
 
         if (view != null) {

